lms/static/sass/course/wiki/_wiki.scss

```diff
@@ -278,30 +278,6 @@ section.wiki {
           color: $link-color;
           font-weight: bold;
           background-color: $gray-l4;
-
-          .icon-view,
-          .icon-home {
-            background-position: -25px 0;
-          }
-
-          .icon-edit {
-            background-position: -25px -25px;
-          }
-
-          .icon-changes,
-          .icon-time {
-            background-position: -25px -49px;
-          }
-
-          .icon-attachments,
-          .icon-file {
-            background-position: -25px -73px;
-          }
-
-          .icon-settings {
-            background-position: -25px -99px;
-          }
-
```

Comment:
Need to replace the sequence slider arrows (left and right), which are still both images.